### PR TITLE
Fix emby-scroller focus and scroll in TV layout

### PR DESCRIPTION
**Pull request by MrTimscampi**
**Changes**
When moving out of an emby-scroller, keep the last focused element, then reuse it as the focus target when entering back into that scroller.

This prevent the weird shifting and inconsistent behavior when moving between scrollers.

This also removes the weird center focus and makes the scolling follow the focus (So going right when at the right edge of the screen will scroll one item to the right, going left at the left edge of the screen will scroll one item left)

**Issues**

### CODE REVIEW

File: src/elements/emby-scroller/emby-scroller.js [hunk 1]
```diff
@@ -17,9 +17,32 @@ import 'css!./emby-scroller';
 
     function initCenterFocus(elem, scrollerInstance) {
         dom.addEventListener(elem, 'focus', function (e) {
-            const focused = focusManager.focusableParent(e.target);
-            if (focused) {
-                scrollerInstance.toCenter(focused);
+            if (e.explicitOriginalTarget.parentNode === e.target.parentNode) {
```
Comment (Camc314) [inline src/elements/emby-scroller/emby-scroller.js:20]: `explicitOriginalTarget` isn't supported in chrome?

Comment (MrTimscampi) [reply src/elements/emby-scroller/emby-scroller.js:20]: Ah, shit... I need the previous target to get this to work... Maybe `relatedTarget` could work...

Comment (dkanada) [reply src/elements/emby-scroller/emby-scroller.js:20]: @Camc314 did you want to adopt this pull request?

File: src/libraries/scroller.js [hunk 2]
```diff
@@ -831,6 +841,10 @@ scrollerFactory.prototype.to = function (location, item, immediate) {
         item = undefined;
     }
 
+    if (location === 'end') {
+        console.warn(this._pos[location]);
```
Comment (thornbill) [inline src/libraries/scroller.js:845]: do we need to keep this?

**Pull request closed without merging**